### PR TITLE
Mar3hc/feat/multiple connect

### PR DESCRIPTION
**Pull request by trimai3001**

### CODE REVIEW

**Review by test-fullautomation — CHANGES_REQUESTED** [review]
Hi @trimai3001 ,
very good!
the release_info file is missing. Please add.
What about doip keyword library?
Thank you,
 Thomas

**Comment by trimai3001** [comment]
Hi @test-fullautomation,
I added release_info file.
I am updating doip keyword library.
Thank you,
Tri

File: config/robotframework_aio/release_items_RobotFramework_UDS.json [hunk 1]
```diff
@@ -97,6 +97,12 @@ Example test case with UDS:
 
 Please refer to `RobotFramework_UDS documentation <https://github.com/test-fullautomation/robotframework-uds/blob/develop/RobotFramework_UDS/RobotFramework_UDS.pdf>`_
 for more details.
+"
+                                       ],
+                   "0.13.1." : [
+"
+* Updated the UDS keywords to support working with multiple connections.
```
Comment (test-fullautomation) [inline config/robotframework_aio/release_items_RobotFramework_UDS.json:104]: Hi @trimai3001 ,
can you please also add the "please refer to ...." from above.
Are we still compatible to already existing test cases? As long as there is only one connection `device` is not required. Did you try running old tests in order to assure that there are no effects?
Thank you,
 Thomas

**Review by test-fullautomation — COMMENTED** [review]
Hi Tri, 
please check my finding and questions.
Thank you,
 Thomas

File: config/robotframework_aio/release_items_RobotFramework_UDS.json (hunk 1 above)
Comment (trimai3001) [reply config/robotframework_aio/release_items_RobotFramework_UDS.json:104]: Hi @test-fullautomation,
I will add the "please refer to..." as requested.
Regarding compatibility with the existing test cases:  I have run the old tests to ensure that there are no side effects, and everything is functioning as expected.
Thank you, 
Tri

Comment (test-fullautomation) [reply config/robotframework_aio/release_items_RobotFramework_UDS.json:104]: very good!

Comment (test-fullautomation) [reply config/robotframework_aio/release_items_RobotFramework_UDS.json:104]: Hi @trimai3001 ,
is the PDF-documentation updated? I see only a robot file with examples how to use.
PDF documentation is missing?
Thank you,
 Thomas

**Review by test-fullautomation — CHANGES_REQUESTED** [review]

**Review by test-fullautomation — APPROVED** [review]
Hi Tri,
very good!
Thank you,
 Thomas

**Pull request merged**